### PR TITLE
Admin: Orders, changed queryset filter

### PR DESCRIPTION
**Pull request by volodymyr-code**
Refs REAL-83

### CODE REVIEW

**Comment by tulimaki** [comment]
@volodymyr-code where is this coming from? Maybe we could have separate view for this and allow superusers and staff to only see orders per shop by default (this is the logic for everything else ATM).

And secondly does the shop filter work nicely. What if there is 1000 shops is that select2 or do we have lag for render? Maybe the shop filter should could be based on name instead select if not already?

Basically what I am after is that we always show data per currently active shop. And we currently just missing the admin modules allowing cross shop data. And these likely should be separated.

**Comment by volodymyr-code** [comment]
> @volodymyr-code where is this coming from? Maybe we could have separate view for this and allow superusers and staff to only see orders per shop by default (this is the logic for everything else ATM).
That task from @chessbr that fix required for one of our project
Christian, please discuss

File: shuup/admin/modules/orders/views/detail.py [hunk 1]
```diff
@@ -28,6 +28,12 @@ class OrderDetailView(DetailView):
     def get_toolbar(self):
         return OrderDetailToolbar(self.object)
 
+    def get_queryset(self):
+        qs = super(OrderDetailView, self).get_queryset().exclude(deleted=True)
+        if not self.request.user.is_superuser:
+            qs = qs.filter(shop=self.request.shop)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/detail.py]: never use `request.shop` in admin views, this is wrong. always use `get_shop(request)`

File: shuup/admin/modules/orders/views/list.py [hunk 2]
```diff
@@ -64,7 +64,10 @@ def get_toolbar(self):
         return toolbar
 
     def get_queryset(self):
-        return super(OrderListView, self).get_queryset().exclude(deleted=True).filter(shop=self.request.shop)
+        qs = super(OrderListView, self).get_queryset().exclude(deleted=True)
+        if not self.request.user.is_superuser:
+            qs = qs.filter(shop=self.request.shop)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/list.py]: use `get_shop`

File: shuup/admin/modules/orders/views/detail.py (hunk 1 above)
Comment (tulimaki) [reply shuup/admin/modules/orders/views/detail.py]: Got it old codes here in diff. Carry on.
